third_party/mbedtls/repo/include/mbedtls/aes.h

```diff
@@ -30,7 +30,7 @@
 #endif
 
 #include <stddef.h>
-#include <stdint.h>
+#include <openthread-std-types.h>
```

Comment:
These changes (changing all references to `<stdint.h>` to `<openthread-std-types.h>`) are not acceptable.

An acceptable alternative would be to create a file named `stdint.h` in the folder `src/replacements/stdint/`. This folder would be included in the system path on systems without `stdint.h`.